app/models/physical_server_provision_task.rb

```diff
@@ -0,0 +1,23 @@
+class PhysicalServerProvisionTask < MiqProvisionTask
+  include_concern 'StateMachine'
+
+  def description
+    'Provision Physical Server'
+  end
+
+  def self.base_model
+    PhysicalServerProvisionTask
```

Comment:
This is the change @lfu @gmcculloug that introduces standalone task name instead shared one (was `miq_provision_task`, is `physical_server_provision_task` now). Not sure why I didn't do it in first place, sorry.